lib/utils.php

```diff
@@ -16,6 +16,11 @@
  * @return array Containing a set of css rules.
  */
 function gutenberg_experimental_get( $array, $path ) {
```

Comment:
I was thinking of what you said about type checking. My view is that type checking is essential complexity to the tools we use and that it needs to be absorbed somewhere. By absorbing it at the helper function level the rest of the code is quite simple. I like it :) 

However, one thing we could do is to perhaps make this even more similar to [lodash's get](https://lodash.com/docs/4.17.15#get):

> gutenberg_experimental_get( $array, $path, $default = array() )

The `$default` will be returned when the `$array` and/or `$path` aren't the expected inputs (now it returns null in the last case). I like that this makes it more explicit. What do you think? Does this lessen your concerns?

Reply:
Ah, I've also noticed that the PHPDoc comment `* @return array Containing a set of css rules.` could use some improvement.

Reply:
First, rewrite WordPress in Rust. ;-)

Ok, I like the suggestion, I've updated the code to include a default 👍